src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -276,8 +262,16 @@ private INamedTypeSymbol GetPropertyTypeByOwnerSymbol(INamedTypeSymbol ownerType
 			return definition;
 		}
 
+		private ISymbol? FindProperty(XamlMember xamlMember) => _findPropertyByXamlMember!(xamlMember);
+
 		private INamedTypeSymbol? FindPropertyType(XamlMember xamlMember) => _findPropertyTypeByXamlMember!(xamlMember);
 
+		private ISymbol? SourceFindProperty(XamlMember xamlMember)
+		{
+			var type = FindType(xamlMember.DeclaringType);
+			return _metadataHelper.FindPropertyByOwnerSymbol(type, xamlMember.Name);
+		}
```

Comment:
I don't see the value of memoizing this. Both calls (`FindType` and `FindPropertyByOwnerSymbol`) are already memoized.

I suspect memoizing this is actually slower as it involves dictionary lookup unnecessarily.

Reply:
Was just blndly following the current setup.